Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -121,23 +121,191 @@ public void ResetConfiguration(MixedRealityToolkitConfigurationProfile profile)
         /// <remarks>
         /// Systems can only be registered once by <see cref="Type"/>
         /// </remarks>
-        public static IReadOnlyDictionary<Type, IMixedRealityService> ActiveSystems => activeSystems;
+        public IReadOnlyDictionary<Type, IMixedRealityService> ActiveSystems => new Dictionary<Type, IMixedRealityService>(activeSystems) as IReadOnlyDictionary<Type, IMixedRealityService>;
 
         private static readonly List<Tuple<Type, IMixedRealityService>> registeredMixedRealityServices = new List<Tuple<Type, IMixedRealityService>>();
```

Comment:
Whenever I see global statics being modified, I tend to worry about concurrency - it might be worth leaving a comment stating explicitly that we don't need to lock/guard against concurrency access because all callers will access in a single-threaded fashion.

Reply:
interesting point. the primary goal for removing the statics is consistency and reducing developer confusion (ex: dont i need an instance to be able to query for data)